project/ThreeMonth03/README.md

```diff
@@ -0,0 +1,121 @@
+# NSD Proposal
+
+## Basic Information
+
+I will implement conjugate gradient method as my final project.
+
+Project URL: [https://github.com/ThreeMonth03/Conjugate-Gradient-Method-library](https://github.com/ThreeMonth03/Conjugate-Gradient-Method-library)
+
+
+## Problem to Solve
+
+I want to solve the global minima of the quadratic function by searching the numerical solution. One of the algorithm that can solve the problem is conjugate gradient method.
```

Comment:
Please change to use 79-char lines.  Long lines are hard to review.  See #89 as an example for how to use short lines.